cpp/src/Glacier2/SessionRouterI.cpp

```diff
@@ -50,10 +50,7 @@ namespace Glacier2
 
         optional<IdentitySetPrx> identities(const Current&) final { return _filters->identitiesPrx(); }
 
-        int getSessionTimeout(const Current& current) final
-        {
-            return static_cast<int>(_sessionRouter->getSessionTimeout(current));
-        }
+        int getSessionTimeout(const Current&) final { return static_cast<int>(_sessionRouter->getSessionTimeout()); }
```

Comment:
we should update this to use `std::int32_t` like the corresponding generated code. It is unfortunate that we use long and int for different timeouts.